data/sql/updates/pending_db_world/rev_1643734354433050200.sql

```diff
@@ -0,0 +1,33 @@
+INSERT INTO `version_db_world` (`sql_rev`)
+VALUES ('1643734354433050200');
+
+UPDATE creature_template
+SET gossip_menu_id=61026
+WHERE entry = 3567;
```

Comment:
Use backticks around the column and table names https://www.azerothcore.org/wiki/sql-standards#general-standards

This line can also be done in one line